### ISSUE TITLE
Allow user to do own transpiling

### ISSUE THREAD

**Issue by matthewp** [stealjs/steal-tools#354]
This way they can use any version of Traceur or Babel or whatever they need to use.

```js
stealTools.build({
  config: __dirname + "/package.json"
}, {
  transpile: function(source) {
    return require('babel-core').compile(source);
  }
});
```

**Comment by whitecolor** [comment]
Yes. it should be! Better not only for js.

**Comment by matthewp** [comment]
@whitecolor There is no transpiling step for non-JavaScript.  CSS, for example, goes through the plugin phase. So any custom stuff you want to do there is with plugins.

**Comment by whitecolor** [comment]
Ok, right, but can babel for build be either replaced as you pointed here https://github.com/stealjs/steal/issues/546#issuecomment-172944874
```
"system": {
  "paths": {
    "babel": "path/to/my/babel.js"
  }
}
```
?

**Comment by matthewp** [comment]
No, transpiling is different and doesn't happen in the build the same way it happens in the browser. All translate plugins do though.

I'm going to try and get this in for you this week.

**Comment by whitecolor** [comment]
Thanks)

**Comment by matthewp** [comment]
@whitecolor This is included in 0.13.3, documented here: http://stealjs.com/docs/steal-tools.BuildOptions.transpile.html

**Comment by whitecolor** [comment]
@matthewp Cool)
